Makefile.defs

```diff
@@ -32,15 +32,26 @@ CMAKE=cmake
 
 # set this to your ARM cross compiler:
 # set CC CXX AR LD RANLIB
-ifneq (,$(filter $(TARGET), kindle arm-generic))
-    export KINDLE=1
+ifeq ($(TARGET), arm-generic)
     CHOST?=arm-linux-gnueabi
-else ifeq ($(TARGET), kindle5)
+else ifeq ($(TARGET), kindle)
     export KINDLE=1
-    CHOST?=arm-kindle5-linux-gnueabi
+    HAS_KINDLE5_TC:=$(shell command -v arm-kindle5-linux-gnueabi-gcc 2> /dev/null)
```

Comment:
If I understand @NiLuJe 's suggestion correctly both this and the Kobo check would be better using something along these lines (untested):

```
ifeq ($(shell gcc -dumpmachine), arm-kindle5-linux-gnueabi)
```

(It's cleaner looking than the `command -v >/dev/null` surroundings in any case)

Reply:
Possibly even w/ $(CC) or whatever we're actually calling instead of hardcoding gcc ;).

Reply:
Good point! We also support clang, albeit mainly as an oddity for Mac OS. :-P

(It may or may not compile slightly faster, which would be conducive to testing, but the actual binaries are slightly slower. Also it might output clearer or at least different error messages when something fails to compile.)

PS Clang actually listens to gcc because it's a drop-in replacement. It pretends to be something like gcc 4.3.